core/plugins/openstack/__init__.py

```diff
@@ -287,6 +287,19 @@ def neutron_bind_interfaces(self):
 
         return interfaces
 
+    @property
+    def octavia_bind_interfaces(self):
+        """
+        Fetch interface o-hm0 used by Openstack Octavia. Returned dict is
+        keyed by config key used to identify interface.
+        """
+        interfaces = {}
+        port = self.nethelp.get_interface_with_name('o-hm0')
+        if port:
+            interfaces.update({'o-hm0': port})
```

Comment:
Is there are an actual config key name we can use instead of "o-hm0"?

Reply:
o-hm0 is hardcoded in charm-octavia https://opendev.org/openstack/charm-octavia/src/branch/master/src/lib/charm/openstack/octavia.py#L39